packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec

```
@@ -50,14 +50,13 @@ BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 7.8.0
 BuildRequires: %{?scl_prefix}npm(@redhat-cloud-services/frontend-components) >= 2.5.0
 BuildRequires: %{?scl_prefix}npm(@redhat-cloud-services/frontend-components) < 3.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 4.14.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 8.16.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 8.17.0
```

you can remove the upper border open as we used to handle it before, e.g:

```suggestion
BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 8.16.0
```

---

@ShimShtein this is still open